int8 agent_number
std_msgs/Bool takeoff_command
